examples/energy-management-app/energy-management-common/device-energy-management/include/DEMDelegate.h

```diff
@@ -19,5 +19,12 @@
 #pragma once
 
 #include <DeviceEnergyManagementDelegateImpl.h>
+#include <DeviceEnergyManagementManager.h>
 
 chip::app::Clusters::DeviceEnergyManagement::DeviceEnergyManagementDelegate * GetDEMDelegate();
+// TODO investigate this why isn't in in DEM?
+extern std::unique_ptr<chip::app::Clusters::DeviceEnergyManagementManager> gDEMInstance;
+extern std::unique_ptr<chip::app::Clusters::DeviceEnergyManagement::DeviceEnergyManagementDelegate> gDEMDelegate;
```

Comment:
A general comment - lots of global variables. However there is an accessor function GetDEMDelegate for the DEM delegate singleton but no accessor functions for the DEM instance. 

Also not sure why gDEMDelegate is being exposed when there is a function to access it?